src/components/comments.js

```diff
@@ -1,33 +1,36 @@
-import React from 'react';
+import React from "react";
 
 import { StaticQuery, graphql } from "gatsby";
 
-import { Disqus } from 'gatsby-plugin-disqus'
+import { Disqus } from "gatsby-plugin-disqus";
 
 function Comments({ data, title, location }) {
-  const disqusConfig = {
-    url: `${data.site.siteMetadata.siteUrl + location}`,
-    identifier: title,
-    title: title,
-  }
-  return (
-    <Disqus config={disqusConfig} />
-  )
+    if (data.site.siteMetadata.disqus === null) return <></>;
```

Comment:
i would prefer returning a null here, but this is fine too

Reply:
I haven't tested it, but I believe we need to return some components? (`React.Fragment` or `<></>` for components.

Reply:
you can take a look at [this](https://stackoverflow.com/questions/42083181/is-it-possible-to-return-empty-in-react-render-function).